examples/testy_bots.py

```diff
@@ -0,0 +1,505 @@
+"""
```

Comment:
Can you look into merging this with everlasting bots?

Reply:
leaving for future PR, but intend to merge functionality into this file